serveradmin/powerdns/models.py

```diff
@@ -17,7 +17,8 @@ class Domain(models.Model):
     id = models.IntegerField(primary_key=True)
     name = models.CharField(max_length=255, null=False)
     master = models.CharField(max_length=128, default=None)
-    type = models.CharField(max_length=6, null=False, choices=TYPES)
+    type = models.CharField(
+        max_length=6, null=False, choices=TYPES, default='NATIVE')
```

Comment:
`max_length` is required and not optional see [Django documentation](https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.CharField.max_length).

You are right about the fact that it is internally handled as a constraint in PostgreSQL and there is no performance impact.

We wouldn't need a new migration as this is not a managed model and the schema is provided by PowerDNS.

Reply:
I see .. forgot that those stuff is unmanaged. Then it is actually quite unimportant .. could maybe still use `models.TextField` then to avoid the need of change if something longer than `NATIVE` comes in.
But, not really important then actually.